pkg/multiplexer/multiplexer.go

```diff
@@ -21,6 +21,25 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+type ErrTimeout struct {
```

Comment:
Looking at the errors on https://pkg.go.dev/net I wonder if `TimeoutError` might be more consistent with standard error naming conventions in Go.

Reply:
That's a good point. I don't remember what tool generated it, but at some point a linter was recommending to prefix errors with `Err`.

I like `TimeoutError` better either way, so I'll go with that. 👍